core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -119,27 +135,40 @@ oppia.factory('SimpleEditorManagerService', [
         data.questionList.getBindableQuestion(
           stateName).setBridgeHtml(newHtml);
       },
-      addNewQuestion: function() {
+      addNewQuestion: function(type) {
         // This effectively adds a new multiple-choice interaction to the
         // latest state in the chain.
-        var lastStateName = (
-          data.questionList.isEmpty() ?
-          SimpleEditorShimService.getInitStateName() :
-          data.questionList.getLastQuestion().getDestinationStateName());
 
-        SimpleEditorShimService.saveInteractionId(
-          lastStateName, DEFAULT_INTERACTION.ID);
-        SimpleEditorShimService.saveCustomizationArgs(
-          lastStateName, DEFAULT_INTERACTION.CUSTOMIZATION_ARGS);
-        SimpleEditorShimService.saveDefaultOutcome(lastStateName, {
-          dest: lastStateName,
-          feedback: [''],
-          param_changes: []
-        });
-
-        var stateData = SimpleEditorShimService.getState(lastStateName);
-        data.questionList.addQuestion(QuestionObjectFactory.create(
-          lastStateName, stateData.interaction, ''));
+          var lastStateName = (
+            data.questionList.isEmpty() ?
+            SimpleEditorShimService.getInitStateName() :
+            data.questionList.getLastQuestion().getDestinationStateName());
+
+            if(type == "multiple"){
```

Comment:
Formatting/spacing: `if (type == 'multiple') {`

Define named constants (QUESTION_TYPE_MULTIPLE_CHOICE, QUESTION_TYPE_CHECKBOX) instead of using the raw strings 'multiple' etc. Use these constants wherever these values are needed.